cmd/proximo-server/main.go

```diff
@@ -63,6 +63,12 @@ func main() {
 		EnvVar: "PROXIMO_DEBUG",
 	})
 
+	configFile := app.String(cli.StringOpt{
+		Name:   "config",
+		Desc:   "Config file",
+		EnvVar: "PROXIMO_CONFIG",
```

Comment:
I would make the name more generic here
```suggestion
		Name:   "acl-config",
		Desc:   "ACL Config file",
		EnvVar: "PROXIMO_ACL_CONFIG",
```